whittle/modules/embedding.py

```diff
@@ -6,6 +6,8 @@
 
 
 class Embedding(torch.nn.Embedding):
+    """An extension of PyTorch's `torch.nn.Embedding` with support of sub-network dimensionality."""
```

Comment:
Maybe rephrase as "An extension of PyTorch's `torch.nn.Embedding` with support  to sub-sample weights corresponding to the sub-network dimensionality"